aws-iot-scheduledaudit/src/main/java/com/amazonaws/iot/scheduledaudit/HandlerUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.amazonaws.iot.scheduledaudit;
+
+import software.amazon.awssdk.services.iot.IotClient;
+import software.amazon.awssdk.services.iot.model.IotException;
+import software.amazon.awssdk.services.iot.model.ListTagsForResourceRequest;
+import software.amazon.awssdk.services.iot.model.ListTagsForResourceResponse;
+import software.amazon.awssdk.services.iot.model.Tag;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HandlerUtils {
```

Comment:
This method can be reused for all of our resources, doesn't seem like "HandlerUtils" specific to ScheudledAudits is the right place for it,